Wrappers/Python/cil/plugins/tigre/FBP.py

```diff
@@ -53,21 +53,9 @@ class FBP(DataProcessor):
 
     def __init__(self, image_geometry=None, acquisition_geometry=None, **kwargs):
```

Comment:
same as above, do we need `kwargs`?

Reply:
kwargs used for device currently

Reply:
discussed with @gfardell and we decided to leave the kwargs in for now